src/sentry/static/sentry/app/views/usageStats/usageStatsProjects.tsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import moment from 'moment';
+
+import AsyncComponent from 'app/components/asyncComponent';
+import LoadingIndicator from 'app/components/loadingIndicator';
+import {Panel, PanelBody} from 'app/components/panels';
+import {DataCategory, Organization} from 'app/types';
+
+import {ProjectUsageStats} from './types';
+
+type Props = {
+  organization: Organization;
+  dataCategory: DataCategory;
+  dataCategoryName: string;
+} & AsyncComponent['props'];
+
+type State = {
+  projectStats: ProjectUsageStats;
+} & AsyncComponent['state'];
+
+class UsageStatsProjects extends AsyncComponent<Props, State> {
+  getEndpoints(): ReturnType<AsyncComponent['getEndpoints']> {
+    const {organization} = this.props;
+
+    // TODO(org-stats): Allow user to pick date range
+    const fourWeeksAgo = moment().subtract(31, 'days').unix();
+    const today = moment().unix();
+
+    return [
+      [
+        'orgStats',
+        `/organizations/${organization.slug}/stats_v2/projects/`,
+        {
+          query: {
+            start: fourWeeksAgo,
+            end: today,
+            interval: '1d',
+          },
+        },
+      ],
+    ];
+  }
+
+  renderError(e: Error) {
+    return (
+      <Panel>
+        <PanelBody>UsageStatsProjects has an error: {e.message}</PanelBody>
+      </Panel>
+    );
+  }
+
+  renderLoading() {
+    return (
+      <Panel>
+        <PanelBody>
+          <LoadingIndicator />
+        </PanelBody>
+      </Panel>
+    );
+  }
+
+  renderBody() {
```

Comment:
You could override `renderComponent` as your loading, error and done states have the same logic.

Reply:
Keeping it as-is for now. I'll amend this when working on the table to display project stats.